### DISCUSSION TITLE
MAUI Example of Bluetooth Classic

### DISCUSSION THREAD

**Discussion by ghost** [inthehand/32feet#428] [Q&A]
Hello, do you have a small sample to use your library to connect android MAUI app, with Bluetooth Classic? I've an arduino device with HC-05 module, that if I send it "Dx"+<CR> where x is the number of door should respond "open" or "close"; but Ican't make it work in maui, on other languages/platforms work(flutter as example), but for this project I need to stay on MAUI. Can you help me please?

**Comment by ramiss** [comment]
Bump. Looking for the same

**Comment by peterfoot** [comment]
I do need to update them as the Maui sample only does discovery at the moment, but you can borrow the code from the Xamarin sample - https://github.com/inthehand/32feet/blob/main/Samples/RfcommXamarinForms/RfcommXamarinForms/MainPage.xaml.cs which shows how to read/write over RFComm
